src/rust/bitbox02-rust/Cargo.toml

```diff
@@ -27,7 +27,8 @@ doctest = false
 
 [dependencies]
 bitbox02 = {path = "../bitbox02"}
-binascii = { version = "0.1.4", default-features = false, features = ["encode"] }
+bitbox02-noise = {path = "../bitbox02-noise"}
+binascii = "0.1"
```

Comment:
Hm... why did this change?

Reply:
wrong rebase i assume from when I bumped binascii in the other PR. will fix.